Sources/Utility/PkgConfig.swift

```diff
@@ -282,12 +282,16 @@ struct PkgConfigParser {
         }
         
         var it = line.characters.makeIterator()
+        // Indicates if we're in a quoted fragment, we shouldn't append quote.
+        var inQuotes = false
         while let char = it.next() {
-            if char == "\\" {
+            if char == "\"" {
```

Comment:
Couldn't parse Libs field into an argument vector: Text ended before matching quote was found for ". (The text was '-L"/usr/local/bin" -l"gt"k 3" -wantareal\\here -one\\ -two')